### PR TITLE
understand-twemproxy-woes

### PR DESCRIPTION
**Pull request by jhwillett**
Exploring https://github.com/ProsperWorks/redis-ick/issues/3 and https://github.com/twitter/twemproxy/issues/537.

Changes the format of the extra `keys` information which was added to ICKSTATS in 0.0.4 (https://github.com/ProsperWorks/redis-ick/pull/2) so that it is compatible with Twemproxy.

Upgrades TravisCI tests so we always test w/ both Redis and Twemproxy=>Redis.

Bumps redis-ick to 0.0.5.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,3 +24,6 @@ script:
   - bundle exec rubocop --version
   - bundle exec rubocop
   - bundle exec env REDIS_URL=redis://localhost:6379 rake test
+  - nutcracker --test-conf --conf-file=.travis/nutcracker.yml
+  - nutcracker             --conf-file=.travis/nutcracker.yml &
+  - bundle exec env REDIS_URL=redis://localhost:22121 rake test
```
Comment (jhwillett) [inline .travis.yml]: I am finding Travis to be a more pliable medium than I expected, which is gratifying.

File: lib/redis/ick.rb [hunk 2]
```diff
@@ -476,7 +476,8 @@ def _eval(lua,ick_key,*args)
       local ick_cset_size = redis.call('ZCARD',ick_cset_key)
       local ick_stats     = {
         'key',        ick_key,
-        'keys',       { ick_key, ick_pset_key, ick_cset_key },
+        'pset_key',   ick_pset_key,
+        'cset_key',   ick_cset_key,
```
Comment (jhwillett) [inline lib/redis/ick.rb]: This nested array was what aggravates Twemproxy and causes https://github.com/ProsperWorks/redis-ick/issues/3 and https://github.com/twitter/twemproxy/issues/537.

File: .rubocop.yml [hunk 3]
```diff
@@ -141,7 +144,9 @@ Style/TernaryParentheses:
 # I am a huge fan of using trailing commas when I break an argument
 # list down one-per line.
 #
-# As such, I reject this test.
+# As such, I reject these tests.
 #
+Style/TrailingCommaInArguments:
```
Comment (achou-prosperworks) [inline .rubocop.yml]: Isn't this cop for a final trailing comma in a method's arguments, per https://rubocop.readthedocs.io/en/latest/cops_style/#styletrailingcommainarguments ?  Which I would want the cop to whistle about.

Comment (jhwillett) [reply .rubocop.yml]: Yes, you are correct about what Style/TrailingCommaInArguments does.

We differ, in that I like the final trailing comma in a multi-line list of arguments.

**Pull request merged**